src/renate/updaters/avalanche/model_updater.py

```diff
@@ -276,6 +276,7 @@ def __init__(
         deterministic_trainer: bool = defaults.DETERMINISTIC_TRAINER,
         gradient_clip_val: Optional[float] = defaults.GRADIENT_CLIP_VAL,
         gradient_clip_algorithm: Optional[str] = defaults.GRADIENT_CLIP_ALGORITHM,
+        mask_unused_classes: bool = defaults.MASK_UNUSED_CLASSES,
```

Comment:
maybe a warning somewhere that this is currently not supported?

Reply:
avalanche?

Reply:
Logged a warning. Is a `warnings.warn` preferred?

Reply:
they have different semantic meanings. warnings.warn is for warnings that can be fixed by the user, i.e., by changing the input. the warning from logging is for warnings that the user is not responsible for.